recipes/modeltest-ng/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.1.5" %}
+
+package:
+  name: modeltest-ng
+  version: {{ version }}
+
+source:
+  url: https://github.com/ddarriba/modeltest/files/2287188/modeltest-ng-0.1.5.tar.gz
```

Comment:
This could use the {{ version }} var.

Reply:
Since I don't know from where come the `2287188`, I didn't find useful to include the var.